src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -83,7 +83,7 @@
             </template>
         </modal>
         <reset-confirmation
-            v-if="showConfirmation"
+            v-if="!dataExplorationMode && showConfirmation"
```

Comment:
The `ResetConfirmation` component appears here, in `ManageFile` and in `FileUpload`. These are the only 3 places I think we need this flag.

Reply:
Except the component itself references the possibly non-existent modules `project` and `stepper` via `mapAction` and `mapState` helpers, so even mounting it throws an error.

Reply:
But I suppose the conditional logic I put in here (https://github.com/mrc-ide/hint/pull/595/files#diff-6f2399eff5f0ba4c4c5783235ba412ffc39c74db05af0349ca5d986e30237312R210) could live in the computed vars of `ResetConfirmation` itself